test/service.thrift

```diff
@@ -0,0 +1,7 @@
+
+service Foo {
+    byte foo(0: byte number) throws (
```

Comment:
I get a warning with the following warning if I try to generate code using the Thrift compiler with a field ID 0 in a struct:

    [WARNING:.../service.thrift:44] Nonpositive value (0) not allowed as a field key.
    [WARNING:.../service.thrift:44] No field key specified for foo, resulting protocol may have conflicts or not be backwards compatible!